Update analytics docs (#794)
hasura/ddn-docs: docs/observability/model-analytics/operation.mdx

This helps in identifying usage patterns, most and least frequently accessed portions of your data graph. In turn
helping optimize performance, manage resources better, and identify potential areas for improvement or scaling.

> Operation analytics can be accessed via the `Analytics` tab within the `Explorer` panel for any specific model or command, allowing users to view data for either an entire project or a single build

Reviewer: ```suggestion
Operation analytics can be accessed via the `Analytics` tab within the `Explorer` panel for any specific model or command, allowing users to view data for either the project API or of a single build
```
